web/templates/stream_settings/stream_settings_overlay.hbs

```diff
@@ -13,7 +13,7 @@
                     <div id="add_new_subscription">
                         {{#if can_create_streams}}
                         <button class="create_stream_button create_stream_plus_button tippy-zulip-delayed-tooltip" data-tooltip-template-id="create-new-stream-tooltip-template" data-tippy-placement="bottom">
-                            <span class="create_button_plus_sign">+</span>
+                            <i class="create_button_plus_sign zulip-icon zulip-icon-square-plus" aria-hidden="true"></i>
```

Comment:
Slightly outside of scope here, but with `aria-hidden` on the icon (which is reasonable), I wonder if the button shouldn't have a translated `aria-label` of "Create new channel" or some such. Could be a follow up.

Reply:
Ah yeah probably. There are several other `zulip-icon-square-plus` icons with `aria-hidden` (which I copied these from) that would also benefit from that. This would be a great first contributor issue -- could you open an issue for that?

Reply:
I feel like I want to look at whether there's a way to rework our model to avoid needing both tooltips and aria-label; probably worth some #frontend discussion.